test.js

```diff
@@ -62,9 +62,25 @@ callbackFunction = function(response){
   });
 };
 
+// Parse the incoming text so that each word is read as a token.
+// This will break the incoming text message into a series of tokens
+// in a list. A message would be broken as follows:
+// Input: "The quick brown fox ".
+// Output: {["The"] ["quick"] ["brown"] ["fox"]}
 // Check to see if we should send the message; this
 // is done by checking for certain keywords.
 function checkForCommand(inText){
+	
+	//If the message does not have "hey bob" (with any caps) just ignore it right away
+	if (req.body.inText.toLower.indexOf("hey bob") = -1)
```

Comment:
Wow, I can't leave a note where no change has occured. Fine. I'm talking about line 30, where the code is:

if(req.body.name != "Bob")

and replace that with: 

if(req.body.[text|message|whatever].toLower.indexOf("hey bob") == -1){

It would be from that point forward that you know it's a command you're looking at.